pkg/utils/yaml.go

```diff
@@ -0,0 +1,13 @@
+package utils
+
+import (
+	"gopkg.in/yaml.v2"
```

Comment:
I think we should move to .v3

Reply:
I'm having a hard time finding any info on v3.  It looks like that main yaml repo only has v2 released? https://github.com/go-yaml/yaml

Reply:
https://gopkg.in/yaml.v3

which links to code in branch v3
https://github.com/go-yaml/yaml/tree/v3

Reply:
Really not sure why master is still v2

Reply:
Quick looks shows the spec looks almost the same as v2.  I feel like I want to wait until they actually cut a release before moving but I'm good either way.